content/en/docs/Tasks/creating-an-index.md

```diff
@@ -5,37 +5,100 @@ description: >
+### Add a bundle to the index
 
-```bash
-$ opm index add --bundles quay.io/my-container-registry-namespace/my-manifest-bundle:0.0.2 --from-index quay.io/my-container-registry-namespace/my-index:1.0.0 --tag quay.io/my-container-registry-namespace/my-index:1.0.1
+```sh
+$ opm render quay.io/example-inc/example-operator-bundle:v0.1.0 \
+    --output=yaml > example-operator-index/index.yaml
 ```
 
-### Other operations on an Index using `opm` 
+Let's validate again:
+```sh
+$ opm validate example-operator-index
+```
+
+Success! There were no errors and we got a `0` error code.
+
+In the general case, adding a bundle involves three discreet steps:
+- Render the bundle into the index using `opm render <bundleImage>`
+- Add the bundle into desired channels and update the channels' upgrade edges
+  to stitch the bundle into the correct place.
+- Validate the resulting index.
+
+> NOTE: Index metadata should be stored in a version control system (e.g. `git`) and index images should be rebuilt from source
+whenever updates are made to ensure that all index changes are auditable.
+
+**Step 1** is just a simple `opm render` command.
+
+**Step 2** has no defined standards other than that the result must pass validation in step 3. Some operator authors may
+decide to hand edit channels and upgrade edges. Others may decide to implement automation (e.g. to idempotently
+build semver-based channels and upgrade graphs based solely on the versions of the operators in the package). There is
+no right or wrong answer for implementing this step as long as `opm validate` is successful.
+
+There are some guidelines to keep in mind though:
+- Once a bundle is present in an index, you should assume that one of your users has installed it. With that in mind,
+  you should take care to avoid stranding users that have that version installed. Put another way, make sure that
+  all previously published bundles in an index have a path to the current/new channel head.
+- Keep the semantics of the upgrade edges you use in mind. `opm validate` is not able to tell you if you have a sane
+  upgrade graph. To learn more about the upgrade graph of an operator, checkout the
+  [creating an upgrade graph doc][upgrade-graph-doc]
+
+### Build and push the index image
+
+Lastly, we can build and push our index:
+
+```sh
+$ docker build . \
+    -f example-operator-index.Dockerfile \
+    -t quay.io/example-inc/example-operator-index:latest
+$ docker push quay.io/example-inc/example-operator-index:latest
+```
 
-The `opm index` command contains additional sub-commands that can be used to perform different operations like remove an operator from an index, prune an index of all but specified operators etc. Please checkout the documentation under `opm index -h` for more information. 
- 
+The resulting image is referred to as an `Index`. Now that image is available for clusters to use and reference with
```

Comment:
Likely overkill given how often it's mentioned by link to the existing "Index" glossary term?